.github/workflows/go.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     name: Build (Windows)
     runs-on: windows-2022
     needs: test
-    if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v')
+#   if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v')
```

Comment:
I reviewed this patch. Now I want to approve this pull request.
So please restore this comment out.